main.go

```diff
@@ -249,20 +263,36 @@ func main() {
 		return
 	}
 	projectID := readProjectID()
+	if projectID == "" {
+		fmt.Println("Could not find any GCP Projects")
+		return
+	}
 	proxyType := readProxyType()
 	cluster := readCluster(projectID)
+	if cluster == nil {
+		fmt.Println("Could not find any GCP Clusters")
+		return
+	}
 	if proxyType == "POD" {
 		loadingStart("Loading Cluster credentials")
-		gcloud.GetClusterCredentials(projectID, cluster)
+		gcloudGetClusterCredentials(projectID, cluster)
 		loadingStop()
 		namespace := readNamespace()
+		if namespace == "" {
```

Comment:
In which case do we return an empty string as namespace? Wouldn't it be nicer to return `nil` so that it is consistent with `readCluster()` and `readPod()`? The same applies to `readProjectID()`.

Reply:
In an empty case.

`readNamespace` returns `string`, not `* string`. Proposed solution would require refactoring those functions. Which can be done in another PR.